pbj-core/gradle.properties

```diff
@@ -1,5 +1,5 @@
 # Version number
-version=0.8.7-SNAPSHOT
+version=0.9.0-SNAPSHOT
```

Comment:
The `0.9.0` introduced maps. You're introducing more new stuff. I suggest to bump up the version number. I don't have a strong opinion if this should be 0.9.1 or 0.10.0 given that your changes don't really break any existing APIs.